modules/deploy/pages/deployment-option/cloud/monitor-cloud.adoc

```diff
@@ -0,0 +1,71 @@
+
+== Configure monitoring
+
+You can monitor Redpanda with Prometheus or with any other monitoring and alerting tool, such as Datadog, New Relic, Elastic Cloud, Google, or Azure.
+
+[NOTE]
+====
+To maximize performance, Redpanda exports some metrics only when the underlying feature is in use. For example, a metric for consumer groups, xref:reference:public-metrics-reference.adoc#redpanda_kafka_consumer_group_committed_offset[`redpanda_kafka_consumer_group_committed_offset`], is only exported when groups are registered.
+====
+
+=== Configure Prometheus
+
+To monitor a Dedicated or BYOC cluster in https://prometheus.io/[Prometheus^]:
+
+. On the Redpanda Cloud *Overview* page for your cluster, under *How to connect*, click the *Prometheus* tab. 
+
+. Click the copy icon for *Prometheus YAML* to copy the contents to your clipboard. 
++
+image::shared:cloud_metrics.png[How to connect screenshot]
++
+The YAML contains the Prometheus scrape target configuration, as well as authentication, for the cluster.
++
+```
+- job_name: redpandaCloud-sample
+  static_configs:
+    - targets:
+        - console-321f60a4.cn6b1srsj6j1mdqdhil0.fmc.ppd.cloud.redpanda.com
+  metrics_path: /api/cloud/prometheus/public_metrics
+  basic_auth:
+    username: prometheus
+    password: "%kvjQr^E!xQ+lXqJ~g?5HhZ3"
+  scheme: https
+```
+
+. Save this information to Prometheus, and observe in Prometheus that metrics from Redpanda endpoints are scraped.
+
+TIP: For a sample Docker Compose file that brings up Prometheus and Grafana, see https://github.com/redpanda-data/observability/tree/main/cloud[Redpanda monitoring examples repository^].
+
+=== Configure Datadog
+
+To monitor a Dedicated or BYOC cluster in https://www.datadoghq.com/[Datadog]:
+
+. On the Redpanda Cloud *Overview* page for your cluster, under *How to connect*, click the *Prometheus* tab. 
+
+. Click the copy icon for *Prometheus YAML* to find scrape target configuration, as well as authentication, for the cluster.
+
+. In Datadog, define the `openmetrics_endpoint` URL for that monitored cluster. The integration configuration should look similar to the following:
++
+```
+instances:
+  # The endpoint to collect metrics about Polaris. Replace ORGANIZATION_NAME with the name of your organization.
+  - openmetrics_endpoint: https://console/-<id>.<identifier>.fmc.cloud.redpanda.com/api/cloud/prometheus/public_metrics
+    use_openmetrics: true
+    collect_counters_with_distributions: true
+
+    auth_type: basic
+    username: prom_user
+    password: prom_pass
+```
+
+NOTE: Integration requires the Datadog agent. The agent must be run in a container in your own container infrastructure. Redpanda does not support launching this container inside a Dedicated or BYOC Kubernetes cluster. For more inforamtion, see the https://docs.datadoghq.com/integrations/redpanda/?tab=host[Datadoc documentation] and https://github.com/DataDog/integrations-extras/tree/master/redpanda[Redpanda Datadog integration] 
```

Comment:
Can we update this to be:

> For more information on building a Datadog agent container with the Redpanda integration, see the [...]

For cloud, this is the only way this would work: one ddagent running to monitor the prom endpoint, since the prom endpoint provides a scrape the includes all brokers.

Reply:
@hcoyote please check if I got [this note](https://deploy-preview-352--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/monitor-cloud/#configure-datadog) right about the Datadog agent.